frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -459,7 +459,7 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>({
                             key,
                             group_type_index: type.group_type_index,
                         })}`,
-                    getName: () => capitalizeFirstLetter(aggregationLabel(type.group_type_index).singular),
+                    getName: (group) => group.name,
```

Comment:
I'm sure it's safe to change this to `group.name` since we're not calling `getName` from the taxonomicGroupTypes anywhere else but worth a double check?

Reply:
the getName usage seems fine. Curious why the current implementation doesn't work? Is aggregation singular not returning anything?

Reply:
We're doing a fuse search in the array of group properties in `infiniteListLogic` and mapping each item in that array as `{name: "Organization", item: item }` because of the `getName` method, when it should be `{name: "timezone", item: item}` instead

Reply:
Ah i see